src/CodiceFiscale/Subject.php

```diff
@@ -111,7 +118,7 @@ public function setBirthDate($birthDate)
     /**
      * Get the gender.
      *
-     * @return Returns the gender.
+     * @return string Returns the gender.
```

Comment:
You could remove **Returns** here and use an uppercase **T** for the initial **the**.